rclpy/test/test_node.py

```diff
@@ -838,6 +842,127 @@ def test_node_set_parameters_rejection(self):
         self.assertIsInstance(result[0], SetParametersResult)
         self.assertFalse(result[0].successful)
 
+    def test_node_set_parameters_rejection_list(self):
+        # Declare a new parameters and set a list of callbacks so that it's rejected when set.
+        reject_list_parameter_tuple = [
+            ('reject', True, ParameterDescriptor()),
+            ('non_reject', True, ParameterDescriptor()),
```

Comment:
If you were expecting these parameters with the name "non_reject" to not be rejected, I think it's wrong. The callback `reject_parameter_callback` will reject these because `'reject' in param.name` will be `True`. 

Please change the names of this parameter to not contain the string "reject" (same for the parameter below).